src/css/controls/imports/autostartmute.less

```diff
@@ -12,7 +12,7 @@
 }
 
 .jwplayer.jw-flag-autostart:not(.jw-flag-media-audio) {
-    &:not(.jw-state-buffering):not(.jw-state-error):not(.jw-state-complete) .jw-display,
+    &:not(.jw-state-buffering):not(.jw-state-error):not(.jw-state-complete):not(.jw-state-playing):not(.jw-state-paused) .jw-display,
```

Comment:
By these `:not`s, it seems we're now showing the display buttons in all states. We might not even need this rule anymore. Check and see what happens if you remove it altogether.